_posts/patterns/2015-12-11-tables.md

```diff
@@ -0,0 +1,257 @@
+---
+layout:         pattern
+title:          Tables
+date:           2015-12-11 00:00:00
+
+categories:     patterns
+tags:
+- atomic
+- pattern
+- buttons
+
+slug:           tables
+
+url_styles:     "components/_tables.scss"
+
+description:    A collection of buttons available for using within the edX platform.
+
+info:           Tables are used for displaying information in tabular format, where is useful to be able to examine and compare data or information in columns and rows. edX offers several table styles to provide ways to display most tabular data, including an example that contains the full set of supported semantic elements.
+
+---
+
+<h3 class="hd-6 example-set-hd">Basic</h3>
+<div class="example-set">
+  <table class="table">
+    <tr>
+      <th>First</th>
+      <th>Last</th>
+      <th>Lineage</th>
+    </tr>
+    <tr>
+      <td>Aragorn II</td>
+      <td>Elessar</td>
+      <td>Son of Arathorn</td>
+    </tr>
+    <tr>
+      <td>Bilbo</td>
+      <td>Baggins</td>
+      <td>Bag End</td>
+    </tr>
+    <tr>
+      <td>Meriadoc</td>
+      <td>Brandybuck</td>
+      <td>Buckland</td>
+    </tr>
+    <tr>
+      <td>Treebeard</td>
+      <td>Fangorn</td>
+      <td>Fangorn Forest</td>
+    </tr>
+  </table>
+</div>
+
+<h3 class="hd-6 example-set-hd">Simplified and compact</h3>
+<div class="example-set">
+  <table class="table table-simplified table-cozy">
+    <tr>
+      <th>First</th>
+      <th>Last</th>
+      <th>Lineage</th>
+    </tr>
+    <tr>
+      <td>Aragorn II</td>
+      <td>Elessar</td>
+      <td>Son of Arathorn</td>
+    </tr>
+    <tr>
+      <td>Bilbo</td>
+      <td>Baggins</td>
+      <td>Bag End</td>
+    </tr>
+    <tr>
+      <td>Meriadoc</td>
+      <td>Brandybuck</td>
+      <td>Buckland</td>
+    </tr>
+    <tr>
+      <td>Treebeard</td>
+      <td>Fangorn</td>
+      <td>Fangorn Forest</td>
+    </tr>
+  </table>
+</div>
+
+<h3 class="hd-6 example-set-hd">Full Semantic Set</h3>
+<div class="example-set">
+  <div class="wrapper-table-scrollable">
```

Comment:
NIT: Would be worthwhile noting that this example is scrollable in the description.